starport/cmd/relayer_connect.go

```diff
@@ -20,83 +20,80 @@ func NewRelayerConnect() *cobra.Command {
 		Short: "Link chains associated with paths and start relaying tx packets in between",
 		RunE:  relayerConnectHandler,
 	}
+
+	c.Flags().AddFlagSet(flagSetKeyringBackend())
+
 	return c
 }
 
-func relayerConnectHandler(cmd *cobra.Command, args []string) error {
-	s := clispinner.New()
-	defer s.Stop()
+func relayerConnectHandler(cmd *cobra.Command, args []string) (err error) {
+	defer func() {
+		err = handleRelayerAccountErr(err)
+	}()
 
-	allPaths, err := xrelayer.ListPaths(cmd.Context())
+	ca, err := cosmosaccount.New(getKeyringBackend(cmd))
 	if err != nil {
 		return err
 	}
 
-	var (
-		givenPathIDs = args
-		allPathIDs   = xstrings.List(len(allPaths), func(i int) string { return allPaths[i].ID })
-		pathsToUse   = xstrings.AllOrSomeFilter(allPathIDs, givenPathIDs)
-	)
+	if err := ca.EnsureDefaultAccount(); err != nil {
+		return err
+	}
 
-	if len(pathsToUse) == 0 {
-		s.Stop()
+	ids := args
 
-		fmt.Println("No chains found to connect.")
-		return nil
-	}
+	s := clispinner.New()
+	defer s.Stop()
+
+	var use []string
 
-	s.SetText("Linking paths between chains...")
+	r := relayer.New(ca)
 
-	linkedPaths, alreadyLinkedPaths, failedToLinkPaths, err := xrelayer.Link(cmd.Context(), pathsToUse...)
+	all, err := r.ListPaths(cmd.Context())
 	if err != nil {
 		return err
 	}
 
-	s.Stop()
-
-	fmt.Println()
-	printSection("Linking chains")
+	// if no path ids provided, then we connect all of them otherwise,
```

Comment:
```suggestion
	// if no path id is provided, then we connect all the paths.
```

Reply:
I used ids because it needs to be plural. I would like to keep the word `otherwise`.